setup.py

```diff
@@ -45,5 +45,9 @@ def readme():
                 'plotly/matplotlylib/mplexporter',
                 'plotly/matplotlylib/mplexporter/renderers'],
       package_data={'plotly': ['package_data/*']},
-      install_requires=['decorator', 'requests', 'six', 'pytz'],
+      install_requires=['decorator',
+                        'nbformat>=4.3',
```

Comment:
oh I see -- it's not on pip yet https://github.com/jupyter/nbformat/issues/90
so our tests were failing with 
```
Collecting nbformat>=4.3 (from plotly==2.0.1)
  Could not find a version that satisfies the requirement nbformat>=4.3 (from plotly==2.0.1) (from versions: 4.0.0, 4.0.1, 4.1.0, 4.2.0)
No matching distribution found for nbformat>=4.3 (from plotly==2.0.1)
```
https://circleci.com/gh/plotly/plotly.py/1942?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link